content/md/assets/work-with-assets.md

```diff
@@ -33,6 +33,32 @@ The following information cannot be sorted out in the asset grid:
 - Tags
 :::
 
+## Asset grid display (_since 2.1_)
+
+To highlight the products' images, a `Gallery` display is available in the asset grid.
```

Comment:
"the assets image" instead of "product's images"

Reply:
effectivement :)